src/main/java/heronarts/lx/parameter/DiscreteParameter.java

```diff
@@ -64,6 +64,7 @@ public DiscreteParameter(String label, int value, int min, int max) {
     super(label, value);
     setRange(min, max);
     setUnits(LXParameter.Units.INTEGER);
+    setWrap(true);
```

Comment:
I think this definitely should not be true by default, as the base case for DiscreteParameters is just an integer value range. Don't think those should all wrap if they slam around the end as behavior at 0 vs. MAX may be *very* discontinuously different.

For EnumParameter I would maybe entertain making it wrappable by default, but think mostly as the precedent is already set here, all should be false by default and can be explicitly set true only where desired.

Reply:
Definitely happy to default everything to wrappable=false.  I only made this one true because I was replacing a previously existing method DiscreteParameter.increment(int) which used a hardcoded wrap=true, and didn't want to muck up anyone's behavior out there if it was in use:

https://github.com/heronarts/LX/pull/55/files#diff-4c24b49cb2e2e475b792bb7a1641236cc4b734d713e6ac97231d1add7e96a6bfL188

Reply:
I'm just going to throw one more "are you sure?" in here.  If a normal expected behavior for DiscreteParameter would be not wrappable, this choice would be causing problems for future people in order to avoid problems for past people.

It could be a good compromise to default DiscreteParameter to wrappable(false) and ObjectParameter to wrappable (true).  The default behavior I would expect on keyboard up/down (and everywhere else) would be for EnumParameter to wrap but not a list of 1-100 integers.

Reply:
Okay cool - yeah I'm agreed, let's kill the keyboard wrapping too, does seem perfectly reasonable not to expect that. Wrappable by default just for ObjectParameter/EnumParameter it is!